transcom/mymove
# B 18112 update source of duty location and pay grade information in profile to show values from orders table

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a871311)

## Summary

This PR is to ensure the orders table information for pay grade and current/origin duty location are shown on the profile and edit profile page, rather than the service member rank and duty location table information.

The edit profile page at this point is still updating the service member rank and duty location before updating the orders table, but the service member rank and current duty location update is no longer driving the order grade and origin duty location update. 

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Access the milmove app as a customer.
2. Create a move and enter in a pay grade and duty location.
3. Put in orders.
4. Navigate to the profile page and ensure that you see pay grade and duty location information that matches the orders table columns of "grade" and "origin_duty_location_id". As a further test, you can edit the service_members rank column to be some other value than the orders grade value and refresh the page to make sure that you are still seeing the orders table value. Same goes for the origin duty location of the orders table.
